src/icrar/leap-accelerate/math/linear_math_helper.cc

```diff
@@ -48,6 +48,19 @@ namespace icrar
         return res;
     }
 
+    std::vector<icrar::MVuvw> ToUVWVector(const Eigen::MatrixXd& value)
+    {
+        auto res = std::vector<icrar::MVuvw>();
+        res.reserve(value.rows());
+
+        for(int row = 0; row < value.rows(); ++row)
+        {
+            auto vector = value(row, Eigen::all);
```

Comment:
Is it possible to receive a reference instead of a copy here? If yes that would be a win.

Reply:
I think this uses a special vector class to store the vector offsets but I can just index the matrix directly instead